src/Roave/SecurityAdvisories/AdvisorySources/GetAdvisoriesFromGithubApi.php

```diff
@@ -80,6 +83,11 @@ public function __invoke(): Generator
             $versions = Type\shape([0 => Type\non_empty_string(), 1 => Type\optional(Type\non_empty_string())])
                 ->assert(Str\split($item['node']['vulnerableVersionRange'], ','));
 
+            if ($item['node']['advisory']['withdrawnAt'] !== null) {
```

Comment:
The return type of `getAdvisories()` needs to be refined a bit: 

```
Cannot access value on variable $item['node'] using offset value of 'advisory', expecting 'vulnerableVersionRange' or 'package' (see https://psalm.dev/115)
            if ($item['node']['advisory']['withdrawnAt'] !== null) {
```

Code all good, IMO - just docblock additions

Reply:
I've resolved the issue in the latest commit.